src/Symfony/Component/Messenger/Stamp/OverrideSendersStamp.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Stamp;
+
+/**
+ * Apply this stamp to change sender to be used instead of the one from configuration file.
+ */
+final class OverrideSendersStamp implements StampInterface
+{
+    private array $senders;
+
+    /**
+     * @param string[] $senders New senders to be used with message
+     */
+    public function __construct(array $senders)
+    {
+        $this->senders = $senders;
+    }
+
+    public function getSenders(): array
```

Comment:
instead of accepting several sender, what about accepting one? multiple senders could still be defined by adding several `ViaSenderStamp` (proposed name :) )

Reply:
My first use case was for simple sender change but I guess it would be healthier to keep multiple senders in same Stamp. I don't have a strong opinion, open for suggestion :)

Reply:
no strong opinion either but if we only look at the stamp, it's less clear to me that adding one `ViaSenderStamp` is going to erase all senders set via the framework config

Reply:
I like it this way.